eureka/S3_data_reduction/niriss.py

```diff
@@ -74,14 +85,12 @@ def read(filename, f277_filename, data, meta):
     meta.time_units = 'BJD_TDB'
 
     # loads all the data into the data object
-    data.data = hdu['SCI',1].data + 0.0
+    data.data = hdu['SCI',1].data * hdu[0].header['EFFINTTM']
     data.err  = hdu['ERR',1].data + 0.0
     data.dq   = hdu['DQ' ,1].data + 0.0
 
-    data.f277 = f277[1].data + 0.0
-
-    data.var  = hdu['VAR_POISSON',1].data + 0.0
-    data.v0   = hdu['VAR_RNOISE' ,1].data + 0.0
+    data.var  = hdu['VAR_POISSON',1].data * hdu[0].header['EFFINTTM']**2.0
+    data.v0   = hdu['VAR_RNOISE' ,1].data * hdu[0].header['EFFINTTM']**2.0
```

Comment:
The code in `b2f.rate2count()` multiplies v0 by time (not time squared).  Can you double check that you really want time squared?

Reply:
removed for now